package.json

```diff
@@ -33,29 +33,137 @@
     "./browser/request.js": "./browser/browser/request.js"
   },
   "types": "./index",
-  "scripts": {
-    "clean": "rimraf lib/* module/* browser/*",
-    "build": "run-s build:transpile build:bundle",
-    "build:transpile": "run-p build:types build:lib build:module build:browser",
-    "build:types": "tsc --emitDeclarationOnly --declaration -p src --outDir lib",
-    "build:lib": "cross-env BABEL_ENV=production npm run build:src -- -d lib",
-    "build:module": "cross-env BABEL_ENV=module npm run build:src -- -d module",
-    "build:browser": "cross-env BABEL_ENV=browser npm run build:src -- -d browser",
-    "build:src": "babel src --extensions \".ts\" --source-maps inline",
-    "build:bundle": "webpack",
-    "lint": "eslint --ext .ts,.js src test",
-    "typecheck": "tsc -p src --noEmit",
-    "pretest:node": "run-s build:transpile",
-    "pretest:browser": "run-s build:browser build:bundle",
-    "pretest:browser-ci": "npm run pretest:browser",
-    "proxy-server": "node node_modules/jsforce-ajax-proxy/lib/server.js",
-    "test": "run-s test:node test:browser",
-    "test:node": "jest --forceExit",
-    "test:browser": "karma start",
-    "test:browser-ci": "concurrently --success first --kill-others \"karma start\" \"npm:proxy-server\""
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:transpile",
+        "build:bundle"
+      ]
+    },
+    "build:transpile": {
+      "dependencies": [
+        "build:node:cjs",
+        "build:browser"
+      ]
+    },
+    "build:node:cjs": {
+      "command": "tsc -p . --pretty --incremental",
+      "files": [
+        "src/**/*.ts",
+        "**/tsconfig.json"
+      ],
+      "output": [
+        "lib/**",
+        "*.tsbuildinfo"
+      ],
+      "clean": "if-file-deleted"
+    },
+    "build:browser": {
+      "command": "babel src --extensions .ts --source-maps inline --env-name browser --out-dir browser",
+      "files": [
+        "src/**/*.ts",
+        "babel.config.js"
+      ],
+      "output": [
+        "browser/**"
+      ]
+    },
+    "build:bundle": {
+      "command": "webpack",
+      "dependencies": [
+        "build:browser"
+      ],
+      "files": [
+        "browser/**/*.js",
+        "webpack.config.js"
+      ],
+      "output": [
+        "dist"
+      ]
+    },
+    "lint": {
+      "command": "eslint --ext .ts src test --color --cache --cache-location .eslintcache",
+      "files": [
+        "src/**/*.ts",
+        "test/**/*.ts",
+        "**/.eslint*"
+      ],
+      "output": []
+    },
+    "test": {
+      "dependencies": [
+        "test:node",
+        "test:browser",
+        "lint"
+      ]
+    },
+    "test:node": {
+      "command": "jest --forceExit",
+      "dependencies": ["build:node:cjs"],
+      "files": [
+        "test/**/*.ts",
+        "jest.setup.js",
+        "jest.config.js"
+      ],
+      "output": []
+    },
+    "test:browser": {
+      "command": "karma start",
+      "dependencies": [
+        "build:bundle"
```

Comment:
karma needs `dist/jsforce.js` to preload in the browser so we generate the browser bundles before.